core-connector/src/main/resources/mappings/getPartiesResponse.ds

```diff
@@ -1,8 +1,23 @@
 local customerDetails = payload.result.customerDetails;
```

Comment:
If any fields, for some reason have changed or are missing in the CBS Finflux response (customerDetails, billDetails, officeName etc.) then the whole transformation will break and code will halt. Also if extensionList is empty [] we wouldn't want to include it in response. This applies to all DataSonnet transformation, as it is quite brittle.